hardware_interface/test/test_component_parser.cpp

```diff
@@ -277,33 +276,31 @@ class TestComponentParser : public Test
       <state_interface name="position"/>
     </joint>
   </ros2_control>
-  <ros2_control name="2DOF_System_Robot_ForceTorqueSensor" type="sensor">
+  <ros2_control name="RRBotForceTorqueSensor2D" type="sensor">
     <hardware>
-      <plugin>ros2_control_demo_hardware/2D_Sensor_Force_Torque</plugin>
+      <plugin>ros2_control_demo_hardware/ForceTorqueSensor2DHardware</plugin>
       <param name="example_param_read_for_sec">0.43</param>
-      <param name="frame_id">kuka_tcp</param>
     </hardware>
     <sensor name="tcp_fts_sensor">
-      <state_interface name="fx">
-        <param name="lower_limits">-100</param>
-        <param name="upper_limits">100</param>
-      </state_interface>
-      <state_interface name="fy">
-        <param name="lower_limits">-100</param>
-        <param name="upper_limits">100</param>
-      </state_interface>
+      <state_interface name="fx"/>
+      <state_interface name="fy"/>
+      <state_interface name="fz"/>
       <state_interface name="tx"/>
       <state_interface name="ty"/>
+      <state_interface name="tz"/>
+      <param name="frame_id">kuka_tcp</param>
```

Comment:
```suggestion
      <param name="frame_id">rrbot_tcp</param>
```

Reply:
This is directly from the roadmap repo.

Reply:
Let's also change there. It doesn't make sense to call this `kuka_tcp`

Reply:
see: ros-controls/roadmap#32

you can change it here or not, as you whish.